compiler/fm/src/file_map.rs

```diff
@@ -30,7 +31,10 @@ impl From<&PathBuf> for PathString {
     }
 }
 #[derive(Debug)]
-pub struct FileMap(SimpleFiles<PathString, String>);
+pub struct FileMap {
+    files: SimpleFiles<PathString, String>,
+    name_to_id: HashMap<PathString, FileId>,
```

Comment:
This is needed so that given a filepath, I can get the FileId assosciated with it?

Reply:
yes, editors don't have fileid information when communicating to server